admin/views/administrator/check-windows-form.ejs

```diff
@@ -0,0 +1,199 @@
+                               value="<%= checkWindowData.adminStartDay %>"
+                               aria-required="true"
+                               aria-label="Administration start day.">
+                    </div>
+                    <div class="form-group form-group-month">
+                        <label class="form-label" for="adminStartMonth">Month</label>
+                        <input class="form-control <%= error.adminStartMonth ? 'form-control-error' : '' %>"
+                               id="adminStartMonth"
+                               name="adminStartMonth"
+                               type="number"
+                               value="<%= checkWindowData.adminStartMonth %>"
+                               aria-required="true"
+                               aria-label="Administration start month.">
+                    </div>
+                    <div class="form-group form-group-year">
+                        <label class="form-label" for="adminStartYear">Year</label>
+                        <input class="form-control <%= error.adminStartYear ? 'form-control-error' : '' %>"
+                               id="adminStartYear"
+                               name="adminStartYear"
+                               type="number"
+                               value="<%= checkWindowData.adminStartYear %>"
+                               aria-required="true"
+                               aria-label="Administration start year.">
+                    </div>
+                </div>
+            </div>
+
+            <hr />
+
+            <div class="form-group">
+                <label class="form-label-bold">
+                    Check start date
+                    <span class="form-hint">For example, 28 4 2017</span>
+                    <% if (typeof error !== 'undefined' &&  error.checkStartDay) { %>
+                    <span class="error-message"><%= error.checkStartDay %></span>
+                    <% } %>
+                    <% if (typeof error !== 'undefined' &&  error.checkStartMonth) { %>
+                    <span class="error-message"><%= error.checkStartMonth %></span>
+                    <% } %>
+                    <% if (typeof error !== 'undefined' &&  error.checkStartYear) { %>
+                    <span class="error-message"><%= error.checkStartYear %></span>
+                    <% } %>
+                </label>
+                <div class="form-date">
+                    <div class="form-group form-group-day">
+                        <label class="form-label" for="checkStartDay">Day</label>
+                        <input class="form-control <%= error.checkStartDay ? 'form-control-error' : '' %>"
+                               id="checkStartDay"
+                               name="checkStartDay"
+                               type="number"
+                               value="<%= checkWindowData.checkStartDay %>"
+                               aria-required="true"
```

Comment:
Good to see the aria-* attributes used here. Perhaps you could come up with documentation to standardise this (in another PR).

Reply:
@jon-shipley Good idea, I'll bring the need for a document during sprint planning.